src/configs/db_config.ts

```diff
@@ -0,0 +1,24 @@
+import { DataSource } from 'typeorm';
+import { User } from '../entity/User';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const OrmConfig = new DataSource({
+  type: 'postgres',
+  host: process.env.DB_HOST || 'localhost',
+  port: parseInt(process.env.DB_PORT || '5432'),
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+
+  synchronize: process.env.NODE_ENV === 'dev' ? false : false,
+
+  logging: process.env.NODE_ENV === 'dev' ? false : false,
+  entities: [User],
```

Comment:
Why are you checking if node_env is dev and assign false to both states ?,
can you try to make that understandable or use different approach on this

Reply:
this file was deleted it turns out it wasnt supposed to be here , we are using ormconfig.js now